.eslintrc.js

```diff
@@ -6,16 +6,16 @@ module.exports = {
   },
   extends: [
     '@nuxtjs/eslint-config-typescript',
-    'plugin:nuxt/recommended'
+    'plugin:vitest/recommended'
```

Comment:
Switching from `plugins` option to `extends` let us not only _enable_ the plugin, but also use its recommended rules automatically.